controllers/ingress_dto.go

```diff
@@ -97,3 +101,9 @@ func pathsV1(rules []networkingv1.IngressRule) []path {
 func viewerFnARN(obj client.Object) string {
 	return obj.GetAnnotations()[cfViewerFnAnnotation]
 }
+
+func originRespTimeout(obj client.Object) int64 {
+	val := obj.GetAnnotations()[cfOrigRespTimeoutAnnotation]
```

Comment:
Here I think we should validate, but I don't found a better way to do it here. I think this validation should be in the controller, but it will need to change this structure too.